canvas_site_deletion/views.py

```diff
@@ -161,3 +174,33 @@ def delete(request, pk):
     messages.success(request, f"Successfully deleted Canvas course {canvas_course_id} and cleaned up course_instance {pk}")
 
     return render(request, 'canvas_site_deletion/index.html')
+
+
+def get_account_hierarchy(course_id) -> list:
+    """
+    Get the list of accounts associated with a given course ID
+    """
+    account_ids = []
+
+    # use course id to get account id
+    canvas_course = canvas_get_course(SDK_CONTEXT, course_id).json()
+    canvas_course_account_id = canvas_course["account_id"] 
+    account_ids.append(canvas_course_account_id)
+
+    # traverse account hierarchy in both directions
```

Comment:
We don't want to traverse the hierarchy in both directions here -- we only want to look up the hierarchy. If we were to also look down the hierarchy, that would mean that a department-level admin would be able to delete a course at the school level which we don't want.